docs/fr/docs/advanced/response-directly.md

```diff
@@ -1,63 +1,63 @@
 
-For example, you cannot put a Pydantic model in a `JSONResponse` without first converting it to a `dict` with all the data types (like `datetime`, `UUID`, etc) converted to JSON-compatible types.
+Par exemple, vous ne pouvez pas mettre un modèle Pydantic dans une `JSONResponse` sans d'abord le convertir en un `dict` avec tous les types de données (comme `datetime`, `UUID`, etc) convertis en types compatibles avec JSON.
 
-For those cases, you can use the `jsonable_encoder` to convert your data before passing it to a response:
+Pour ces cas, vous pouvez spécifier un appel à `jsonable_encoder` pour convertir vos données avant de les passer à une réponse :
 
-```Python hl_lines="6-7  21-22"
-{!../../../docs_src/response_directly/tutorial001.py!}
+``Python hl_lines="6-7 21-22"
+{ !../../../docs_src/response_directly/tutorial001.py!}
 ```
 
-!!! note "Technical Details"
-    You could also use `from starlette.responses import JSONResponse`.
+!!! note "Détails techniques"
+    Vous pouvez aussi utiliser `from starlette.responses import JSONResponse`.
 
-    **FastAPI** provides the same `starlette.responses` as `fastapi.responses` just as a convenience for you, the developer. But most of the available responses come directly from Starlette.
+    **FastAPI** fournit le même objet `starlette.responses` que `fastapi.responses` juste par commodité pour le développeur. Mais la plupart des réponses disponibles proviennent directement de Starlette.
 
-## Returning a custom `Response`
+## Retourner une `Response` personnalisée
 
-The example above shows all the parts you need, but it's not very useful yet, as you could have just returned the `item` directly, and **FastAPI** would put it in a `JSONResponse` for you, converting it to a `dict`, etc. All that by default.
+L'exemple ci-dessus montre toutes les parties dont vous avez besoin, mais il n'est pas encore très utile, car vous auriez pu retourner l'`item` directement, et **FastAPI** l'aurait mis dans une `JSONResponse` pour vous, en le convertissant en `dict`, etc. Tout cela par défaut.
 
-Now, let's see how you could use that to return a custom response.
+Maintenant, voyons comment vous pourriez utiliser cela pour retourner une réponse personnalisée.
 
-Let's say that you want to return an <a href="https://en.wikipedia.org/wiki/XML" class="external-link" target="_blank">XML</a> response.
+Disons que vous voulez retourner une réponse <a href="https://en.wikipedia.org/wiki/XML" class="external-link" target="_blank">XML</a>.
 
-You could put your XML content in a string, put it in a `Response`, and return it:
+Vous pouvez mettre votre contenu XML dans une chaîne de caractères, la placer dans une `Response`, et la retourner :
 
-```Python hl_lines="1  18"
-{!../../../docs_src/response_directly/tutorial002.py!}
+``Python hl_lines="1 18"
+{ !../../../docs_src/response_directly/tutorial002.py!}
 ```
```

Comment:
Same here:

```
```Python hl_lines="1  18"
{!../../../docs_src/response_directly/tutorial002.py!}
```

Reply:
I correct this, if it's ok for you, please, close the comment. Thanks.